### DISCUSSION TITLE
CodeFirst时怎么设置数据库列的默认值或绑定

### DISCUSSION THREAD

**Discussion by topstop3416** [dotnetcore/FreeSql#1072] [Q&A]
[Column(IsNullable = false, InsertValueSql = "''")]
InsertValueSql 是通过FreeSql插入才有效 怎么设置数据库本身的默认值 不通过FreeSql插入也有效呢

![sqlserver默认值](https://user-images.githubusercontent.com/37021873/163720528-e67fc380-0a3a-4278-bb1f-3922d6e8613b.png)

**Comment by 2881099** [comment]
数据库的默认值不支持
